simplified-ui-settings/src/main/java/org/nypl/simplified/ui/settings/SettingsFragmentDebug.kt

```diff
@@ -45,42 +43,54 @@ import org.nypl.simplified.ui.toolbar.ToolbarHostType
 import org.slf4j.LoggerFactory
 
 /**
- * A fragment that shows the set of accounts in the current profile.
+ * A fragment that shows various debug options for testing app functionality at runtime.
```

Comment:
Oops. Embarassing that that stayed in here for this long...

Reply:
Just a little copy pasta 🍝